skimage/util/noise.py

```diff
@@ -201,15 +210,14 @@ def random_noise(image, mode='gaussian', seed=None, clip=True, **kwargs):
         out = image.copy()
         p = kwargs['amount']
         q = kwargs['salt_vs_pepper']
-        flipped = _bernoulli(p, image.shape)
-        salted = _bernoulli(q, image.shape)
+        flipped = _bernoulli(p, image.shape, seed)
+        salted = _bernoulli(q, image.shape, seed)
```

Comment:
_bernoulli expects `rng` rather than seed? I think that is causing some of the test failures